.github/workflows/changelog.yml

```diff
@@ -69,3 +69,23 @@ jobs:
           key: ${{ runner.os }}-go-pkg-mod-${{ hashFiles('go.sum') }}
       - run: cd tools && go install github.com/client9/misspell/cmd/misspell
       - run: misspell -error -source text CHANGELOG.md
+
```

Comment:
I'm wondering if we should add a check to ensure the changelog file naming (number) matches the PR number.

Reply:
I can think of a few cases where this won't necessarily be true, so it might be best to wait and see how much of a problem it is:

* Separate pull request to add a changelog entries for previously merged pull requests
* Pull request to fix an existing changelog entry

Reply:
Thinking about this more, it might be really good if we separately had _content_ checking tooling to reduce some maintainer burden:

* Release note blocks should not contain new lines (instead should be multiple release note blocks)
* Ensuring category entries except features contain entry categories
* If there is an entry category, ensure it has the prefix `resource/aws_`, `data-source/aws_`, or `provider`

Doesn't need to be part of the MVP though.